### PR TITLE
Replace DISPLAY_[WIDTH|DEPTH|HEIGHT] by display->[width|height|depth] and HAGL_CHAR_BUFFER_SIZE conditional define

### PR DESCRIPTION
**Pull request by CHiPs44**

### CODE REVIEW

**Comment by CHiPs44** [comment]
I replaced DISPLAY_XXXX in README.md and I shouldn't have removed the section that was duplicated.

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,6 @@ modules.order
 Module.symvers
 Mkfile.old
 dkms.conf
+
+.vscode
```
Comment (tuupola) [inline .gitignore:54]: This belongs to your global .gitignore. The project .gitignore should only contain language specific ignores.

Comment (CHiPs44) [reply .gitignore]: Cleaned up.

**Comment by CHiPs44** [comment]
Thanks!

**Pull request merged**